notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "b6dbb8aa-c070-4e7d-8b7a-64588e7f0da4",
```

Comment:
<p>maybe really better to move this to notebooks_utils?</p><p> </p><p>I think it may confuse users as it is just some notebook related stuff, not quantization process related</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Copied the util file from #1365